src/Analysis/Abstract/Graph.hs

```diff
@@ -70,17 +68,14 @@ graphingTerms :: ( Member (Reader ModuleInfo) sig
                  , Member (Reader (CurrentFrame address)) sig
                  , Member (Reader (CurrentScope address)) sig
                  , Member (Reader ControlFlowVertex) sig
-                 , VertexDeclaration syntax
-                 , Declarations1 syntax
+                 , VertexDeclaration term
                  , Ord address
-                 , Foldable syntax
-                 , term ~ Term syntax Loc
                  , Carrier sig m
                  )
-              => Open (term -> Evaluator term address value m a)
-graphingTerms recur term@(Term (In a syntax)) = do
```

Comment:
Abstract parsing requires that we abstract over the term types, not the syntax types.